zerver/lib/events.py

```diff
@@ -441,6 +441,8 @@ def name(sub):
     elif event['type'] == "update_display_settings":
         if event['setting_name'] == "twenty_four_hour_time":
             state['twenty_four_hour_time'] = event["setting"]
+        if event['setting_name'] == "high_contrast_mode":
+            state['high_contrast_mode'] = event["setting"]
```

Comment:
OK I just merged 61f0778296fc58f4fd31125e3deb34987560ca11 to eliminate the need to change this section of the code when adding new display settings.

@sarahcstringer FYI :)